test/e2e/tests/transaction/send-eth.spec.js

```diff
@@ -155,21 +153,27 @@ describe('Send ETH', function () {
       await withFixtures(
         {
           fixtures: new FixtureBuilder().build(),
-          ganacheOptions: {
-            ...defaultGanacheOptions,
+          anvilOptions: {
             hardfork: 'london',
           },
           smartContract,
           title: this.test.fullTitle(),
+          localNode: 'anvil',
         },
-        async ({ driver, ganacheServer }) => {
-          await logInWithBalanceValidation(driver, ganacheServer);
+        async ({ driver, contractRegistry, anvilServer }) => {
+          const contractAddress = await contractRegistry.getContractAddress(
+            smartContract,
+          );
+          await logInWithBalanceValidation(driver, anvilServer);
 
           // Wait for balance to load
           await driver.delay(500);
 
           await driver.clickElement('[data-testid="eth-overview-send"]');
-          await driver.clickElement({ text: 'Account 1', tag: 'button' });
```

Comment:
this was a bug in the test. We are supposed to test a send to a multisig contract, but here we selected the account 1 as recipient. Now this is fixed